### PR TITLE
nodejs interact with api example

### PR DESCRIPTION
**Pull request by noamtcohen**

### CODE REVIEW

File: site/source/docs/porting/connecting_cpp_and_javascript/Interacting-with-code.rst [hunk 1]
```diff
@@ -186,6 +186,46 @@ in :ref:`preamble-js`.
...
+
+.. code:: bash
+
+    emcc api_example.c
+
+Require the library and call it's procedures from node:
+
+.. code:: javascript
+
+    var em_module = require('./a.out.js');
```
Comment (kripken) [inline site/source/docs/porting/connecting_cpp_and_javascript/Interacting-with-code.rst]: do you need `./` here? i wonder if the example might be more general without it (might run on windows).

**Comment by kripken** [comment]
Nice, thanks! Please add yourself to `AUTHORS`, and squash the commits into one.

File: site/source/docs/porting/connecting_cpp_and_javascript/Interacting-with-code.rst (hunk 1 above)
Comment (noamtcohen) [inline site/source/docs/porting/connecting_cpp_and_javascript/Interacting-with-code.rst]: yes, this is needed and shouldn't be a problem on windows, node will normalise this.

**Comment by noamtcohen** [comment]
Where do I add myself to `AUTHORS`?
Squash the commits... is that a `rebase` or `reset`?...

**Comment by kripken** [comment]
The `AUTHORS` file is in the top level directory.

You can squash commits with `git rebase -i emscripten/incoming` (or however the remote is called instead of `emscripten/`), then replace for all commits after the first, replace `pick` with `squash` (or just `s`).

File: site/source/docs/porting/connecting_cpp_and_javascript/Interacting-with-code.rst (hunk 1 above)
Comment (kripken) [inline site/source/docs/porting/connecting_cpp_and_javascript/Interacting-with-code.rst]: Oh ok, good.

**Comment by noamtcohen** [comment]
I messed-up   the repo and couldn't understand how to clean it up so I just re-forked and created a new pull request: https://github.com/kripken/emscripten/pull/4059

**Pull request closed without merging**